src/modules/person/domain/person.ts

```diff
@@ -45,15 +43,15 @@ export class Person<WasPersisted extends boolean> {
         public auskunftssperre?: boolean,
     ) {
         this.state = { passwordReset: false };
-        this.newPasswordInternal = this.password;
+        this.newPasswordInternal = this.password ?? 'unset';
```

Comment:
Was this a decision that was made? I feel like an unset password should not have a string value (especially since this is basically a magic string)

Reply:
I already got rid of that actually. It should now be simply undefined

Reply:
84898285f44c022279d0cc704bcd812aebbd0e50